docs/guide/execution/quickrun_execution.rst

```diff
@@ -0,0 +1,42 @@
+.. _userguide_quickrun:
+
+Execution with Quickrun
+=======================
+
+The planning and preparation of a campaign of alchemical simulations using the ``openfe`` package is intended to be
+achievable on a local workstation in a matter of minutes.
+The **execution** of these simulations however requires a large amount of computational power and is intended to be
+distributed across a HPC environment.
```

Comment:
It's probably not necessary, but one thing here that might be a bit of a catch to a novice user is "o so I need a HPC machine to do this", maybe a note that says something like "you can also do this on your workstation" might be good to spell things out?